docs/src/pages/components/data-grid/selection/DisableRowSelection.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { DataGrid, GridRowParams } from '@material-ui/data-grid';
+import { randomUserName, randomInt } from '@material-ui/x-grid-data-generator';
+
+const columns = [
+  { field: 'id' },
+  { field: 'username', width: 150 },
+  { field: 'age', width: 80, type: 'number' },
+];
+
+const rows = [
```

Comment:
The other demos on this page are using `import { useDemoData } from '@material-ui/x-grid-data-generator';
`. Would it be simpler?

Reply:
The ID value in `useDemoData` is a GUID, we can't use it for the odd/even demo. What do you think in disabling rows where the status is "Filled" or other condition? https://material-ui.com/components/data-grid/demo/

Reply:
Not real preferences, what we already have is good enough.